src/pages/partners.tsx

```diff
@@ -140,8 +144,15 @@ export default function partners(): ReactElement {
 						<TabPanels mt={10}>
 							<TabPanel>
 								<SimpleGrid
-									minChildWidth="250px"
-									spacing="40px"
+									columns={[2, null, 3]}
+									spacing="20px"
```

Comment:
```suggestion
									spacing={6}
```
Try to use Chakra-native spacings, not pixels